tests/rest/media/test_media_retention.py

```diff
@@ -267,23 +267,23 @@ def _assert_if_mxc_uris_purged(
         def _assert_mxc_uri_purge_state(mxc_uri: MXCUri, expect_purged: bool) -> None:
             """Given an MXC URI, assert whether it has been purged or not."""
             if mxc_uri.server_name == self.hs.config.server.server_name:
-                found_media_dict = self.get_success(
-                    self.store.get_local_media(mxc_uri.media_id)
+                found_media = bool(
```

Comment:
Casting to a boolean was easier than trying to tell mypy that it was a union of two different types, especially when we just care if a result was returned for tests.

Reply:
Thanks. (Casting would be my preference, anyway.)